### DISCUSSION TITLE
是否可以添加GitHub Action支持？

### DISCUSSION THREAD

**Discussion by myp0402** [YunYouJun/hexo-theme-starry#311] [Q&A 疑难杂症] [answered]
如题，每次更改都要重新部署都太麻烦了:(

**Comment by YunYouJun** [comment, marked as answer]
GitHub Actions 应该本来就是支持的……

你是想要一个最简单的例子吗？

我认为文档已经提供了一个示例。

https://yun.yunyoujun.cn/guide/#%E6%8C%81%E7%BB%AD%E9%9B%86%E6%88%90
